include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -858,13 +1080,27 @@ __pattern_min_element(__hetero_tag<_BackendTag>, _ExecutionPolicy&& __exec, _Ran
 }
 
 #if _ONEDPL_CPP20_RANGES_PRESENT
+
+template <typename _Comp, typename _Proj>
+struct __pattern_min_element_comp
```

Comment:
We already defined a type with the same functionality above. And we can re-use one.

Reply:
Fixed